RefApp/ViewControllers/AssetListTableViewController.swift

```diff
@@ -109,6 +108,54 @@ extension AssetListTableViewController {
     }
 }
 
+
+// MARK: - TableView Delegate
+extension AssetListTableViewController {
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        
+        if assets[indexPath.row].assetId != "" {
```

Comment:
I think `guard` instead of `if` demonstrates the purpose more clearly here

Reply:
Yes agree, guard should be use there